Map + Job Naming Changes

Realtimeslayer opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Adjusts Autopsy office, medical locker room, saferoom atmospherics, and minor changes to job naming.

## Why It's Good For The Game

Having a less confusing medical layout, and slightly more functional atmospherics system, as well as more logic to job naming is better in my opinion.

## Proof of Testing

Check unit tests for this.

## Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs to prevent unnecessary GBP loss. You can read up on GBP and its effects on PRs in the tgstation guides for contributors. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->

:cl:
map: shifted medical locker room location
map: added more atmospherics to medical saferoom
code: changed some job names to be a little more logical
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Realtimeslayer committed:
Changes some job names

Realtimeslayer committed:
map changes

see PR for full

Realtimeslayer committed:
fixings

Realtimeslayer committed:
fixing air tanks

DirectorAutumn reviewed (changes requested):
Please provide screenshots of the new layout

Realtimeslayer commented:
<img width="390" height="651" alt="image" src="https://github.com/user-attachments/assets/ea862029-4642-43bf-845f-2b9cda174678" />
^ where old med lockers were located, moved and replaced with an auxiliary atmospherics system, like on all the other decks.
<img width="566" height="327" alt="image" src="https://github.com/user-attachments/assets/77700c7f-b4ad-4a6c-b344-57f4caafbb32" />
^ new location of the med lockers, a little less confusing to find.
<img width="529" height="483" alt="image" src="https://github.com/user-attachments/assets/3bf96187-010f-4439-b778-8cf820347688" />
^ autopsy back to original position

DirectorAutumn commented:
Otherwise the rest looks good @Realtimeslayer

Realtimeslayer commented on code/modules/mob/living/carbon/human/human.dm:
```diff
@@ -243,19 +243,17 @@ var/global/list/rank_prefix = list(
-	"Emergency Response Unit" = "Emergency Response Unit", \
 	"Counselor" = "Doctor", \
 	"Psychologist" = "Counselor", \
-	"Peacekeeper" = "Sergeant", \
```
Not *everybody* would be a Sergeant however, so having it be some name like Operative or something such as that would be a little better, and just call them by their actual rank which they have set.

Realtimeslayer replied:
Several ranks are available, though they are all Petty Officers so I might just set there to be no name? Since having PO3 and PO2 in front of your name is weird. Also ranks are on IDs, and character examine I think if ID is equipped.

DirectorAutumn reviewed (dismissed):
All good

Realtimeslayer committed:
update

Realtimeslayer committed:
Merge branch 'dev' into additions
